src/ui/components/ServerLaunchSection.css

```diff
@@ -20,6 +20,9 @@
 #playButton .BigButton {
   height: 100%;
   box-shadow: 0 0 5px 3px rgba(0, 0, 0, 0.2);
+  font-size: 24px;
+  font-weight: 500;
+  width: 20%;
```

Comment:
This seems to cause the launch button to match the size of the other icon-labelled buttons, which should be smaller:
![image](https://github.com/Grasscutters/Cultivation/assets/25207995/77bd6173-31e9-490b-983d-aeffed72d037)

Original sizes:
![image](https://github.com/Grasscutters/Cultivation/assets/25207995/2d68ad0f-a889-4c3a-a438-e17eb677cabc)

The launch button should always be the largest button, as it should be the "main focus", in a sense. Not everyone is starting their own server every time (some not ever), and not everyone wants to open the extras window every time, so they should not have equal real estate.